TODO

```diff
@@ -32,6 +32,11 @@ Janitorial Clean-ups:
 
 Features:
 
+* drop nss-myhostname in favour of nss-resolve?
+
```

Comment:
I don't think so. Not everybody is using resolve and not everybody *should* be using it. For example in aa mock chroot I don't want systemd-resolve to run, but I still want `localhost` to be resolvable. `myhostname` provides useful functionality that is partially independent of `resolve` and I think we should keep it, because of this, and because of backwards compatibility.